.github/workflows/vast.yaml

```diff
@@ -235,13 +235,14 @@ jobs:
           run_if_changed_default docker-compose $run_docker_compose "docker/" "Dockerfile" ".dockerignore"
           run_docker_vast="$(any ${run_docker_compose} ${run_vast_io} ${run_python})"
           run_if_changed_default docker-vast $run_docker_vast
+          run_if_changed_default regression-tests $run_docker_vast
           run_if_changed vast-nix "$VAST_SOURCES[@]" plugins/ contrib/vast-plugins/ flake.nix flake.lock nix/
 
   policy-enforcement:
     name: Policy Enforcement
     needs:
       - configure
-    if: ${{ needs.configure.outputs.run-policy-enforcement == 'true' }}
+    if: github.event_name != 'workflow_dispatch'
```

Comment:
This should be moved back to `run-policy-enforcement` and set from the configuration job.